### DISCUSSION TITLE
Does strongSwan implement RFC 4307?

### DISCUSSION THREAD

**Discussion by JackelynOliveira** [strongswan/strongswan#1130] [Q&A]
I could find a [text file](https://github.com/strongswan/strongswan/blob/master/doc/standards/rfc4307.txt) in your repository mentioning RFC 4307, but I couldn't find it listed on your [documentation](https://docs.strongswan.org/docs/5.9/features/ietf.html). Does strongSwan implement RFC 4307?

**Comment by grahamqs** [comment]
That RFC is ancient now, why would you want to use old ciphers ?

4307 has been obseleted by 8247.. You might want to look at 8247.
